src/app/notifications/notifications.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngFor="let notification of allNotifications$ | async">
+  <div class="container text-center notification-bar {{ notification.priority }}">
+    <div class="container-fluid text-center" *ngIf="!notificationDismissed(notification)" data-cy="notification-banner">
+      {{ notification.message }}
```

Comment:
You should use the markdown component so that any markdown in the message will be rendered correctly.

Reply:
Done.